src/main/java/org/javacord/bot/listeners/CommandCleanupListener.java

```diff
@@ -24,8 +24,24 @@
      * @return The embed builder for call chaining.
      */
     public static EmbedBuilder insertResponseTracker(EmbedBuilder builder, long trackedMessageId) {
+<<<<<<< HEAD
```

Comment:
...

Reply:
i don't know what happened there lol

Reply:
You had a merge conflict and didn't resolve it properly